docs/portal/transports/edit-transport.md

```diff
@@ -1,11 +1,11 @@
 # Edit a Transport
 
-To edit a transport, search the transport you want to edit in the <strong> Transport </strong> page if it was not in the page, then click the "Action" button and select the "Edit".
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of HDT portal.
+To edit a transport, click the "Action" button of the transport entry in the list and select "Edit". If the transport was not in the current page, you can search it in the search box.
```

Comment:
Another instance of "search it." I suggest replacing all instances of this string with "search for it."